test/end-to-end/journal/trial_balance/trialBalance.page.js

```diff
@@ -27,15 +28,35 @@ function TrialBalancePage() {
     return gridRows.count();
   }
 
-  function openErrorModal() {
-    viewErrorButton.click();
+  function viewErrorList() {
+    return viewErrorButton.click();
+  }
+
+  function resetView(){
+    return resetButton.click();
+  }
+
+  function showAccountDetailInTransaction(n) {
+    const columnNumber = 5;
+
+    const row = grid
```

Comment:
What would be the difference between the `GridTester` class and `GridUtils`?  It seems to me they would do the same thing...